### PR TITLE
Completed css-layout-classes exercise - Julie Dumbrique

### PR DESCRIPTION
**Pull request by juliedumbrique**
Screen Shot 2021-08-19 at 3 03 04 PM

### CODE REVIEW

File: css-layout-classes/index.html [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+      <div class="row">
+        <div class="column">
+          <h2>Donkey Kong</h2>
+          <p>Donkey Kong is a series of video games featuring the adventures of a gorilla called Donkey Kong, conceived by
+            Shigeru Miyamoto in 1981. The franchise consists mainly of two game genres, but also includes additional
+            spin-off titles of various genres.</p>
+          <p>The games of the first genre are the mostly single-screen platform/action puzzle types, featuring Donkey Kong
+            as the opponent in an industrial construction setting.</p>
+        </div>
+        <img src="./images/dk.png" alt="image of dk" class="image">
```
Comment (codingcodymiller) [inline css-layout-classes/index.html]: This `column` and `image` element should have a column class like your other column classes, that describe the styling they apply.

File: css-layout-classes/styles.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+.row {
+  display: flex;
+}
+
+.column {
+  display: flex;
+  flex-direction: column;
+}
```
Comment (codingcodymiller) [inline css-layout-classes/styles.css]: There shouldn't be a need for `flex-direction: column` here, this should be doable with just setting `width` on the elements using `column` classes.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]
Great start here! A few notes for you:

File: css-layout-classes/styles.css [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  margin-right: auto;
+}
+
+.row {
+  display: flex;
+}
+
+.column {
+  display: flex;
+}
```
Comment (codingcodymiller) [inline css-layout-classes/styles.css]: This can probably be removed 🤔

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by juliedumbrique** [comment]
Screen Shot 2021-08-20 at 9 59 27 AM

**Review by codingcodymiller — APPROVED** [review]
Great work!

**Pull request merged**